lua/telescope/_extensions/cheat.lua

```diff
@@ -0,0 +1,86 @@
+local has_telescope, telescope = pcall(require, 'telescope')
+
+if not has_telescope then
+  error('This plugins requires nvim-telescope/telescope.nvim')
+end
+
+local curl = require("plenary.curl")
+local finders = require("telescope.finders")
+local make_entry = require("telescope.make_entry")
+local pickers = require("telescope.pickers")
+local sorters = require("telescope.sorters")
+local conf = require("telescope.config").values
+local previewers = require("telescope.previewers")
+local utils = require('telescope.utils')
+local putils = require('telescope.previewers.utils')
+local actions = require('telescope.actions')
+local defaulter = utils.make_default_callable
+local M = {}
+
+require('telescope').setup {}
+
+M.query = function(qry)
+  local q, ft
+
+  if qry == nil or vim.tbl_isempty(qry) then
+    q = ":list"
+  elseif qry.child == nil then
+    q = string.format("%s/:list", qry.ft)
+    ft = qry.ft
+  else
+    q = string.format("%s/%s", qry.ft, qry.child)
+    ft = qry.ft
+  end
+  return {
+    res = vim.split(curl.get{ url = "cht.sh/"..q.."?T" }.body or "", "\n"),
+    ft = ft or ""
+  }
+end
+
+M.get_content = function(ft, child)
+  return M.query({ft = ft, child = child})
+end
+
+M.cheat_fd = function(opts) -- TODO Make it non blocking
+  opts = opts or {}
+  local res = M.query({ft = opts.ft, child = opts.child})
+  local ft = res.ft
```

Comment:
You don't do anything with that

Reply:
You mean  `local res = query({ft = opts.ft, child = opts.child})` >

Reply:
No with the `local ft = res.ft`